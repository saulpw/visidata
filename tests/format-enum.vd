sheet	col	row	longname	input	keystrokes	comment
			open-file	sample_data/sample.tsv	o	
sample	OrderDate		type-date		@	set type of current column to date
sample	Units		type-int		#	set type of current column to int
sample	Unit_Cost		type-float		%	set type of current column to float
sample			columns-sheet		Shift+C	open Columns Sheet: edit column properties for current sheet
sample_columns	fmtstr	ã‚­OrderDate	edit-cell	%Y	e	edit contents of current cell
sample	Item	0	setcol-format-enum	Pencil=1 Binder=2 Pen=3 Desk=4		add secondary type translator to current column from input enum (space-separated)
