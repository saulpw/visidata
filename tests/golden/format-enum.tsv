OrderDate	Region	Rep	Item	Units	Unit_Cost	Total
2016	East	Jones	1	95	1.99	189.05
2016	Central	Kivell	2	50	19.99	999.50
2016	Central	Jardine	1	36	4.99	179.64
2016	Central	Gill	3	27	19.99	539.73
2016	West	Sorvino	1	56	2.99	167.44
2016	East	Jones	2	60	4.99	299.40
2016	Central	Andrews	1	75	1.99	149.25
2016	Central	Jardine	1	90	4.99	449.10
2016	West	Thompson	1	32	1.99	63.68
2016	East	Jones	2	60	8.99	539.40
2016	Central	Morgan	1	90	4.99	449.10
2016	East	Howard	2	29	1.99	57.71
2016	East	Parent	2	81	19.99	1619.19
2016	East	Jones	1	35	4.99	174.65
2016	Central	Smith	4	2	125.00	250.00
2016	East	Jones	Pen Set	16	15.99	255.84
2016	Central	Morgan	2	28	8.99	251.72
2016	East	Jones	3	64	8.99	575.36
2016	East	Parent	3	15	19.99	299.85
2016	Central	Kivell	Pen Set	96	4.99	479.04
2016	Central	Smith	1	67	1.29	86.43
2016	East	Parent	Pen Set	74	15.99	1183.26
2017	Central	Gill	2	46	8.99	413.54
2017	Central	Smith	2	87	15.00	1305.00
2017	East	Jones	2	4	4.99	19.96
2017	West	Sorvino	2	7	19.99	139.93
2017	Central	Jardine	Pen Set	50	4.99	249.50
2017	Central	Andrews	1	66	1.99	131.34
2017	East	Howard	3	96	4.99	479.04
2017	Central	Gill	1	53	1.29	68.37
2017	Central	Gill	2	80	8.99	719.20
2017	Central	Kivell	4	5	125.00	625.00
2017	East	Jones	Pen Set	62	4.99	309.38
2017	Central	Morgan	Pen Set	55	12.49	686.95
2017	Central	Kivell	Pen Set	42	23.95	1005.90
2017	West	Sorvino	4	3	275.00	825.00
2017	Central	Gill	1	7	1.29	9.03
2017	West	Sorvino	3	76	1.99	151.24
2017	West	Thompson	2	57	19.99	1139.43
2017	Central	Andrews	1	14	1.29	18.06
2017	Central	Jardine	2	11	4.99	54.89
2017	Central	Jardine	2	94	19.99	1879.06
2017	Central	Andrews	2	28	4.99	139.72
